tests/kernel/wave/wave_gemm_test.py

```diff
@@ -137,5 +145,5 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
                 f.write(mb.module_op.get_asm())
 
         iree_ref = torch.zeros(shape[0], shape[1], dtype=torch.float32)
```

Comment:
you also need to add `benchmark_batch_size`, `benchmark_repetitions` and `benchmark_results_file` like in igemm test.

Reply:
I can add that. Are those numbers good?
        config["benchmark_batch_size"] = 10
        config["benchmark_repetitions"] = 3

Reply:
Yes, lets go with these value for now.